frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -172,16 +172,22 @@ export default class TransactionsApi extends ExplorerApi {
           //
           // Once the above TODO is resolved, we should just move this to TransactionInfo method
           // (i.e. query the information there only for the specific transaction).
-          const actions = transactionExtraInfo.transaction.actions;
-
-          transaction.actions = actions.map((action: RpcAction | string) => {
+          const actions = await this.call<any>("select", [
```

Comment:
Well, given the above comments, we cannot do this yet :disappointed: 

You may notice that we store Actions in the DB, but never use them, and you are absolutely right. this is the case now.

Let's keep this code as a comment here, so in future we can simply uncomment it.

Reply:
okaaaaay

Reply:
Well, Wallet does use actions available from the Explorer Backend, so we should also have a PR for Wallet.